src/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.5.1)
```

Comment:
You don't need this here. The top-level file should take care of this for the whole project.